packages/react/src/utils/types.ts

```diff
@@ -72,14 +69,22 @@ type StandardBaseProps = {
       /** @deprecated Use subscriber prop instead */
       subscriberId: string;
       subscriber?: never;
+      applicationIdentifier: string;
     }
   | {
       subscriber: Subscriber | string;
       subscriberId?: never;
+      applicationIdentifier: string;
+    }
+  | {
+      // Keyless mode - no subscriber or subscriberId or applicationIdentifier
+      subscriber?: never;
+      subscriberId?: never;
+      applicationIdentifier?: never;
```

Comment:
fix for the keyless type that was breaking the autosuggestions in the IDE cc @djabarovgeorge